2024-10-29 14:27:34,652 INFO    MainThread:38752 [wandb_setup.py:_flush():79] Current SDK version is 0.18.5
2024-10-29 14:27:34,652 INFO    MainThread:38752 [wandb_setup.py:_flush():79] Configure stats pid to 38752
2024-10-29 14:27:34,652 INFO    MainThread:38752 [wandb_setup.py:_flush():79] Loading settings from C:\Users\gabma\.config\wandb\settings
2024-10-29 14:27:34,653 INFO    MainThread:38752 [wandb_setup.py:_flush():79] Loading settings from C:\Users\gabma\Downloads\Text2Music_Ai\wandb\settings
2024-10-29 14:27:34,653 INFO    MainThread:38752 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-10-29 14:27:34,653 INFO    MainThread:38752 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-29 14:27:34,653 INFO    MainThread:38752 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program_relpath': 'training.py', 'program_abspath': 'C:\\Users\\gabma\\Downloads\\Text2Music_Ai\\training.py', 'program': 'C:\\Users\\gabma\\Downloads\\Text2Music_Ai\\training.py'}
2024-10-29 14:27:34,653 INFO    MainThread:38752 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-29 14:27:34,653 INFO    MainThread:38752 [wandb_setup.py:_flush():79] Applying login settings: {'mode': 'offline'}
2024-10-29 14:27:34,653 INFO    MainThread:38752 [wandb_init.py:_log_setup():534] Logging user logs to C:\Users\gabma\Downloads\Text2Music_Ai\wandb\offline-run-20241029_142734-557aakzs\logs\debug.log
2024-10-29 14:27:34,653 INFO    MainThread:38752 [wandb_init.py:_log_setup():535] Logging internal logs to C:\Users\gabma\Downloads\Text2Music_Ai\wandb\offline-run-20241029_142734-557aakzs\logs\debug-internal.log
2024-10-29 14:27:34,653 INFO    MainThread:38752 [wandb_init.py:init():621] calling init triggers
2024-10-29 14:27:34,653 INFO    MainThread:38752 [wandb_init.py:init():628] wandb.init called with sweep_config: {}
config: {'wav_folder': 'C:\\Users\\gabma\\Downloads\\Text2Music_Ai\\dataset\\wavs', 'csv_file': 'C:\\Users\\gabma\\Downloads\\Text2Music_Ai\\dataset\\metadata.csv', 'hidden_dim': 512, 'time_steps': 1000, 'beta_start': 0.0001, 'beta_end': 0.02, 'batch_size': 1, 'epochs': 50, 'learning_rate': 0.0001, 'weight_decay': 1e-06, 'num_workers': 4, 'log_interval': 10, 'checkpoint_interval': 10, 'checkpoint_dir': 'C:\\Users\\gabma\\Downloads\\Text2Music_Ai\\checkpoints', 'use_wandb': True}
2024-10-29 14:27:34,653 INFO    MainThread:38752 [wandb_init.py:init():671] starting backend
2024-10-29 14:27:34,653 INFO    MainThread:38752 [wandb_init.py:init():675] sending inform_init request
2024-10-29 14:27:34,655 INFO    MainThread:38752 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=spawn, using: spawn
2024-10-29 14:27:34,656 INFO    MainThread:38752 [wandb_init.py:init():688] backend started and connected
2024-10-29 14:27:34,660 INFO    MainThread:38752 [wandb_init.py:init():783] updated telemetry
2024-10-29 14:27:34,661 INFO    MainThread:38752 [wandb_init.py:init():816] communicating run to backend with 90.0 second timeout
2024-10-29 14:27:34,820 INFO    MainThread:38752 [wandb_init.py:init():867] starting run threads in backend
2024-10-29 14:27:34,951 INFO    MainThread:38752 [wandb_run.py:_console_start():2463] atexit reg
2024-10-29 14:27:34,951 INFO    MainThread:38752 [wandb_run.py:_redirect():2311] redirect: wrap_raw
2024-10-29 14:27:34,951 INFO    MainThread:38752 [wandb_run.py:_redirect():2376] Wrapping output streams.
2024-10-29 14:27:34,951 INFO    MainThread:38752 [wandb_run.py:_redirect():2401] Redirects installed.
2024-10-29 14:27:34,953 INFO    MainThread:38752 [wandb_init.py:init():911] run started, returning control to user process
2024-10-29 14:29:28,780 INFO    MainThread:38752 [wandb_run.py:_finish():2158] finishing run music-diffusion/557aakzs
2024-10-29 14:29:28,781 INFO    MainThread:38752 [wandb_run.py:_atexit_cleanup():2426] got exitcode: 0
2024-10-29 14:29:28,781 INFO    MainThread:38752 [wandb_run.py:_restore():2408] restore
2024-10-29 14:29:28,781 INFO    MainThread:38752 [wandb_run.py:_restore():2414] restore done
